src/test/java/org/fundacionjala/coding/HLJoseTorrez/HighestAndLowestTest.java

```diff
@@ -0,0 +1,76 @@
+package org.fundacionjala.coding.HLJoseTorrez;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Created by JoseTorrez on 6/12/2017.
+ */
+public class HighestAndLowestTest {
+
+    /**
+     *
+     */
+    @Test
+    public void someTest() {
+        assertEquals(HighestAndLowest.getHightAndLow("4 5 29 54 4 0 -214 542 -64 1 -3 6 -6"), "542 -214");
```

Comment:
The first argument in the assertEquals is the expectedResult and the second argument is the actual result.
Change it.